app/editor/src/components/form/toning/ToningGroup.tsx

```diff
@@ -0,0 +1,64 @@
+import { useFormikContext } from 'formik';
+import { IContentModel } from 'hooks';
+import React from 'react';
+import { Col, Row } from 'tno-core';
+
+import { Error } from '../error';
+import * as styled from './styled';
+
+export interface IToningGroupProps {
+  fieldName: keyof IContentModel;
+}
+
+export const ToningGroup: React.FC<IToningGroupProps> = ({ fieldName }) => {
+  const toningOptions = [-5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5];
+  const { values, setFieldValue, errors, touched } = useFormikContext<IContentModel>();
+
+  const [active, setActive] = React.useState<number>();
+  React.useEffect(() => {
+    if (values.tonePools?.length && values.tonePools[0].value) setActive(values.tonePools[0].value);
```

Comment:
defaults to `-5`?  Is this what we want?

Reply:
this is actually looking at the existing formik value of tonePools which the way we have it setup will always be tonePools[0]. it currently defaults to nothing when creating new content!